### PR TITLE
CTest updates.

### PR DESCRIPTION
**Pull request by jeffqjiangNew**
- Added VP9 test.
 - Added tests for video decode raw sample.
 - Changed test video streams to raw elementary or IVF formats.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,12 @@ Full documentation for rocDecode is available at [https://rocm.docs.amd.com/proj
 
 * The new bitstream reader feature. The bitstream reader contains a few built-in stream file parsers, including elementary stream file parser and IVF container file parser. Currently the reader can parse AVC, HEVC and AV1 elementary stream files and AV1 IVF container files. More format support will be added in the future.
 * A new sample app, called videodecoderaw which uses the bitstream reader instead of FFMPEG demuxer to get picture data.
+* More core tests.
```
Comment (LakshmiKumar23) [inline CHANGELOG.md]: Change this to 'More CTests' for clarity

Comment (jeffqjiangNew) [reply CHANGELOG.md:11]: What does CTests stand for?

Comment (spolifroni-amd) [reply CHANGELOG.md]: If you can also say which tests, that helps too.

File: README.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ access the video decoding features available on your GPU.
 * H.265 (HEVC) - 8 bit, and 10 bit
 * H.264 (AVC) - 8 bit
 * AV1 - 8 bit, and 10 bit
+* VP9 - 8 bit, and 10 bit
```
Comment (spolifroni-amd) [inline README.md:16]: This information also needs to be added to docs/rocDecode-prerequisites.rst

**Review by spolifroni-amd — CHANGES_REQUESTED** [review]
The changes to the prerequisites need to be reflected in the rst file.

File: CHANGELOG.md (hunk 1 above)
Comment (jeffqjiangNew) [reply CHANGELOG.md]: Done.

File: README.md (hunk 2 above)
Comment (jeffqjiangNew) [reply README.md:16]: Not quite sure why the supported codec type info needs to be in docs/install/rocDecode-prerequisites.rst? @spolifroni-amd

Comment (jeffqjiangNew) [reply README.md:16]: Actually should be rocDecode-formats-and-architectures.rst. Updated.

**Review by AryanSalmanpour — COMMENTED** [review]
The videoDecodeBatch function fails with this pull request. This test is passing with ToT.

CI Pre-checking failure: 

Stage 'PreCheckin: Test rhel9 && gfx1101 - mainline'
image


Stage 'PreCheckin: Test ubuntu22 && gfx942 - mainline'

image

**Review by spolifroni-amd — CHANGES_REQUESTED** [review]
The name of a documentation page cannot be changed at random. There is a process. Please undo this change. Thank you.

File: README.md (hunk 2 above)
Comment (spolifroni-amd) [reply README.md:16]: The page is named rocDecode-prerequisites.rst

Changing the name entails changing redirects etc. And a Jira issue needs to be opened to track it. If you want the name changed, please open a SWDEV Doc issue for it and send me the issue number. 

In the meantime, please change the name back. Thank you.

Comment (jeffqjiangNew) [reply README.md:16]: I'm not sure what this really means. The rocDecode-prerequisites.rst file is in docs/install folder, which does NOT have the supported codec type info. However, in docs/reference/rocDecode-formats-and-architectures.rst, the supported codec types are listed and I added VP9 in this file. The file name is NOT changed. Please review again. @spolifroni-amd

**Comment by jeffqjiangNew** [comment]
> The videoDecodeBatch function fails with this pull request. This test is passing with ToT.
> 
> CI Pre-checking failure:
> 
> Stage 'PreCheckin: Test rhel9 && gfx1101 - mainline' image
> 
> Stage 'PreCheckin: Test ubuntu22 && gfx942 - mainline'
> 
> image

Is isn't this the driver issue we discussed in the meeting yesterday?

**Comment by AryanSalmanpour** [comment]
> > The videoDecodeBatch function fails with this pull request. This test is passing with ToT.
> > CI Pre-checking failure:
> > Stage 'PreCheckin: Test rhel9 && gfx1101 - mainline' image
> > Stage 'PreCheckin: Test ubuntu22 && gfx942 - mainline'
> > image
> 
> Is isn't this the driver issue we discussed in the meeting yesterday?

@jeffqjiangNew  No, it is not a driver issue. The problem was that VP9 was not handled properly in the videoDecodeBatch sample. I have pushed a fix for it in this PR.

**Comment by jeffqjiangNew** [comment]
> The name of a documentation page cannot be changed at random. There is a process. Please undo this change. Thank you.

It is not changed. Please see the above comment.

**Review by AryanSalmanpour — APPROVED** [review]

File: test/CMakeLists.txt [hunk 3]
```diff
@@ -69,23 +69,49 @@ add_test(
...
+add_test(
+  NAME
+    video_decode-VP9
+  COMMAND
+    "${CMAKE_CTEST_COMMAND}"
+            --build-and-test "${ROCM_PATH}/share/rocdecode/samples/videoDecode"
+                              "${CMAKE_CURRENT_BINARY_DIR}/videoDecode"
+            --build-generator "${CMAKE_GENERATOR}"
+            --test-command "videodecode"
+            -i ${ROCM_PATH}/share/rocdecode/video/AMD_driving_virtual_20-VP9.ivf
```
Comment (LakshmiKumar23) [inline test/CMakeLists.txt:108]: do we not have a mp4 file for VP9?

**Review by rrawther — APPROVED** [review]

File: test/CMakeLists.txt (hunk 3 above)
Comment (jeffqjiangNew) [reply test/CMakeLists.txt:108]: I didn't create the mp4 version.

**Review by LakshmiKumar23 — APPROVED** [review]

**Pull request merged**